__tests__/components/Link.spec.js

```diff
@@ -1,15 +1,31 @@
-/* globals expect, test */
+/* globals expect, describe, it */
+import { mount } from 'enzyme'
 import React from 'react'
 import renderer from 'react-test-renderer'
 import Provider from '../../src/components/Provider'
 import Link from '../../src/components/Link'
 
-test('Link renders correctly', () => {
-  const component = renderer.create(
-    <Provider>
-      <Link>Link</Link>
-    </Provider>
-  )
-  let tree = component.toJSON()
-  expect(tree).toMatchSnapshot()
+describe('Link component', () => {
+  it('Should match the stored snapshot', () => {
+    const component = renderer.create(
+      <Provider>
+        <Link>Link</Link>
+      </Provider>
+    )
+    let tree = component.toJSON()
+    expect(tree).toMatchSnapshot()
+  })
+
+  it('Should display the href prop if a child is not provided', () => {
+    const url = 'https://google.com'
+    const component = mount(
+      <Provider>
+        <Link href={url} />
+      </Provider>
+    )
+
+    console.log(component.html())
+
+    expect(component.find(`a [href="${url}"]`)).toHaveLength(1)
```

Comment:
NIT: I think this doesn't really test the spec, as it simply checks if there is an anchor with a `href` set to `url`. I think it should be something like ```expect(component.find(`a [href="${url}"]`)[0].text()).toEqual(url)```

Reply:
@sradevski Yep you're totally right! This was a clear mistake, I'll fix it.